commands/create_cluster.go

```diff
@@ -53,14 +55,16 @@ func defaultAddClusterArguments() addClusterArguments {
 		inputYAMLFile:           cmdInputYAMLFile,
 		numWorkers:              cmdNumWorkers,
 		owner:                   cmdOwner,
+		releaseVersion:          cmdRelease,
 		scheme:                  scheme,
 		token:                   token,
-		releaseVersion:          cmdRelease,
 		wokerAwsEc2InstanceType: cmdWorkerAwsEc2InstanceType,
 		wokerAzureVMSize:        cmdWorkerAzureVMSize,
 		workerNumCPUs:           cmdWorkerNumCPUs,
 		workerMemorySizeGB:      cmdWorkerMemorySizeGB,
 		workerStorageSizeGB:     cmdWorkerStorageSizeGB,
+		workersMax:              cmdWorkersMax,
```

Comment:
The old var was called `numWorkers`, would it make sense to call the new ones `numWorkersMax` accordingly?

Reply:
I would keep it like that for now. There are more important things to get right. My priority here is only to get the feature out of the door, as we have to put more love into this code base anyway.